src/Mailgun/Resource/Api/Suppressions/Bounce/IndexResponse.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * Copyright (C) 2013-2016 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Resource\Api\Suppressions\Bounce;
+
+use Mailgun\Resource\Api\PaginationResponse;
+use Mailgun\Resource\ApiResponse;
+
+/**
+ * @author Sean Johnson <sean@mailgun.com>
+ */
+class IndexResponse implements ApiResponse
+{
+    use PaginationResponse;
+
+    /**
+     * @var Bounce[]
+     */
+    private $items;
+
+    /**
+     * @param Bounce[] $items
+     * @param array    $paging
+     */
+    public function __construct(array $items, array $paging)
+    {
+        $this->items = $items;
+        $this->paging = $paging;
+    }
+
+    /**
+     * @param array $data
+     *
+     * @return Bounce[]
```

Comment:
`Bounce[]` is an argument to create a `IndexResponse`, and in the end you're calling `return new self()` so you're returning an `IndexResponse` object. Could you change it please? Thanks,